src/types/misc.ts

```diff
@@ -13,7 +13,7 @@ import { Range } from "./range";
  * Intellisense from resolving it.
  * See https://github.com/microsoft/TypeScript/issues/31940#issuecomment-841712377
  */
-export type Alias = {};
+export type Alias = {} & {};
```

Comment:
The typing on mouse hover seemse to be workign now in VS code on ly pc :blush: 

While we're there we could take the occasion to correclty type the colors that we missed previously. For example the colors in the cell Style, or all the color constants.

Reply:
added a commit for this.  I might have overlooked some occurences though